firmware/Components.cpp

```diff
@@ -543,8 +460,8 @@ void AtlasPH::onSerial(){
   }
 }
 
-AtlasDO::AtlasDO(int channel) 
-  : Sensor(channel), SerialSensor(channel, 9600), measurementInterval(3000)
+AtlasDO::AtlasDO(int id, int port) 
+  : ExternalSensor(id, port), SerialSensor(id, port, 9600), measurementInterval(3000)
```

Comment:
It might make sense to have `measurementInterval` be a constructor argument with a default value instead of being a constant hidden inside the initializer.